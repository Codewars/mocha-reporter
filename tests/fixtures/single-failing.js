const assert = require("assert");

describe("group 1", () => {
  it("test 1", () => {
    assert.strictEqual(1, 2);
  });
});
