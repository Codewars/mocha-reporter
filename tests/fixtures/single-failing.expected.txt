
<DESCRIBE::>group 1

<IT::>test 1

<FAILED::>Expected values to be strictly equal:<:LF:><:LF:>1 !== 2<:LF:>

<COMPLETEDIN::>1

<COMPLETEDIN::>1
